app-near/src/app/_components/survey/broadcasting/BroadcastingButton.tsx

```diff
@@ -55,13 +56,26 @@ const BroadcastingButton: React.FC<BroadcastingButtonProps> = (
     },
   };
 
+  const getTypeformeBaseUrl = (surveyType: SurveyType) => {
+    switch (surveyType) {
+      case SurveyType.CARBON_FOOTPRINT:
+        return env.NEXT_PUBLIC_TYPEFORM_CARBON_FOOTPRINT_LINK;
+      case SurveyType.SU:
+        return env.NEXT_PUBLIC_TYPEFORM_SU_LINK;
+      case SurveyType.WAY_OF_LIFE:
+        return env.NEXT_PUBLIC_TYPEFORM_WAY_OF_LIFE_LINK;
+      default:
+        break;
+    }
+  };
```

Comment:
:green_circle: Utilisation d'une map à la place d'un switch case ?
```suggestion
const typeformBaseUrls: Record<SurveyType, string | undefined> = {
  [SurveyType.CARBON_FOOTPRINT]: env.NEXT_PUBLIC_TYPEFORM_CARBON_FOOTPRINT_LINK,
  [SurveyType.SU]: env.NEXT_PUBLIC_TYPEFORM_SU_LINK,
  [SurveyType.WAY_OF_LIFE]: env.NEXT_PUBLIC_TYPEFORM_WAY_OF_LIFE_LINK,
};

const getTypeformeBaseUrl = (surveyType: SurveyType) => typeformBaseUrls[surveyType];
```